torch/testing/_internal/common_utils.py

```diff
@@ -5617,6 +5617,36 @@ def remove_cpp_extensions_build_root():
         else:
             shutil.rmtree(default_build_root, ignore_errors=True)
 
+
+def install_cpp_extension(extension_root):
```

Comment:
is this copied from run_test.py?

Reply:
with some changes, yes :) eventually I plan to purge run_test.py of this logic entirely